### PR TITLE
[Stopwatch] Add types to private properties

### PR DESCRIPTION
**Pull request by derrabus**
| Q             | A
| ------------- | ---
| Branch?       | 6.0
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | N/A
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

**Review by Nyholm — APPROVED** [review]
Thank you

File: src/Symfony/Component/Stopwatch/Section.php [hunk 1]
```diff
@@ -21,27 +21,16 @@ class Section
...
-     */
-    private $morePrecision;
-
-    /**
-     * @var string
-     */
-    private $id;
+    private ?float $origin;
+    private bool $morePrecision;
+    private ?string $id = null;
```
Comment (noniagriconomie) [inline src/Symfony/Component/Stopwatch/Section.php:28]: should we default it to `$this->id = '';` inside the constructor?
nullable seems weird :s

Comment (derrabus) [reply src/Symfony/Component/Stopwatch/Section.php:28]: I didn't want to change the behavior here.

Comment (noniagriconomie) [reply src/Symfony/Component/Stopwatch/Section.php:28]: understood :)

**Comment by nicolas-grekas** [comment]
Thank you @derrabus.

**Pull request merged**